enterprise/javaee.project/src/org/netbeans/modules/javaee/project/api/ant/ui/logicalview/AbstractLogicalViewProvider.java

```diff
@@ -526,7 +527,8 @@ public Image getIcon( int type ) {
         public Image getOpenedIcon( int type ) {
             Image original = super.getOpenedIcon(type);
             if (isBroken()) {
-                return ImageUtilities.mergeImages(original, ProjectProperties.ICON_BROKEN_BADGE.getImage(), 8, 0);
+                return ImageUtilities.mergeImages(original,
+                        ImageUtilities.icon2Image(ProjectProperties.ICON_BROKEN_BADGE), 8, 0);
```

Comment:
same question

Reply:
Yup, I'll remove these from the patch, as mergeImages is smart enough to avoid the need.